IndoorMapsFrontend/src/App.css

```diff
@@ -2,3 +2,23 @@
   margin: 0 auto;
   text-align: center;
 }
+
+.everythingContainer {
```

Comment:
nit: A better name would be `outerContainer` or `mainContainer` or `pageWrapper` depending on use-case

Reply:
Change applied: https://github.com/Theo-s-Meta-Capstone/IndoorMaps/commit/d21fe5eb4af3fe4915c728ba95ef5cce28b593c6